docs/data/sql_functions.yml

```diff
@@ -646,6 +646,9 @@ collection:
   - sql: ARRAY_UNION(array1, array2)
     table: haystack.arrayUnion(array)
     description: Returns an array of the elements in the union of array1 and array2, without duplicates. If any of the array is null, the function will return null.
+  - sql: ARRAY_CONCAT(array1, array2, tail...)
+    table: array1.arrayConcat(array1, array2, tail...)
+    description: Returns an array of the elements in the concat of at least two arrays, allow duplicates. If all of the arrays are null, the function will return null.
```

Comment:
In jira it was mentioned that we could do it in BigQuery's way
BigQuery allows having only one array and just returns it back.
Why should we not allow this?